### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by dev-prabhat**
Refactor Landing Page of E-commerce application

image

image

### CODE REVIEW

**Review by purvasharmadev — APPROVED** [review]
code is clean and understandable, and UI is awesome.
good to merge.

**Comment by santhoshpemmaka** [comment]
Looking Good. Great Work 💯 
Good to Merge 👍

**Review by santhoshpemmaka — APPROVED** [review]

File: src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                            </p>
+                        </div>
+                    </div>
+                    <div className="card-img position-rel margin-xs">
+                        <img
+                            className="img-responsive"
+                            src={jacket_collection}
+                            alt="winter_collection"
+                        />
+                        <div className="card-badge-secondary position-abs top-50 left-0 padding-xs d-100 head-sm">
```
Comment (zoef-yt) [inline src/pages/Home.js]: Convert them into a react component and do not repeat the same component.

**Review by zoef-yt — APPROVED** [review]
The code looks good. The UI looks great. Add description for the pr and add a preview link too.

**Pull request merged**